robot_log_visualizer/ui/autogenerated/visualizer.py

```diff
@@ -275,6 +284,7 @@ def retranslateUi(self, MainWindow):
         self.actionQuit.setText(_translate("MainWindow", "&Quit"))
         self.actionQuit.setShortcut(_translate("MainWindow", "Ctrl+Q"))
         self.actionOpen.setText(_translate("MainWindow", "&Open"))
+        self.actionConnect.setText(_translate("MainWindow", "Realtime Connect"))
```

Comment:
Here we can disable the button if yarp / blf is not found In this case we can change the text to `Realtime connect (yarp and blf are required)	`

Reply:
Fixed in [f8b88fa](https://github.com/ami-iit/robot-log-visualizer/pull/80/commits/f8b88fab1f0898a6ebcbe8d74165af6a6f4ff085)